### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by smunozmo**
# Template 5
[Figma Template #5 ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A1095)

Figma's desktop template size: 1440px width. 
Figma's mobile template size: 320px width. 
 
## Updates:

* Created Hello, Projects, About, Contact, and Footer sections.
* Added basic transitions to projects cards without animation.
* Changed px for vw to keep sizes proportional for media query.

No linters errors detected. ✅
Used flex-box for layout. ✅

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,140 +13,188 @@
...
     </header>
-    <main class="name">
-        <h1>Sebastián<br>Muñoz</h1>
+    <main>
+        <div class="main-text">
+            <div class="name">
+                <h1>Sebastián<br>Muñoz</h1>
+            </div>
+            <section class="intro">
+                <p>Hello! I am a software developer! I can help you build a product, feature or website.      Taka a look of my works. <br> If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
```
Comment (MahmoudBakr23) [inline index.html]: - [ ] Please remove the extra indentations between "website.          Take a ...".
Kindly notice that we're only adding blank lines and indentations for good reasons such as separating logical code or two large sections from each other.

> You can read more about this and other useful notes [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

File: css/style.css [hunk 2]
```diff
@@ -331,45 +338,44 @@ textarea {
...
+  visibility: hidden;
 }
 
 /* Contact form end */
 
 footer {
   background-color: #1a2236;
-  width: 320px;
-  height: 150px;
+  width: 100vw;
```
Comment (MahmoudBakr23) [inline css/style.css]: - [ ] Please notice that using the `vw` unit in the `width` property is causing an overflow on the website. Kindly keep in mind that [`vw`](https://stackoverflow.com/questions/25225682/difference-between-width100-and-width100vw) stands for "viewport width" and this means that this element is going to take all the available space on the screen including the document margin. Please use the `%` unit instead or just delete the line (it won't make a difference) in order to fix the design issue.

> Please consider this note in the future and on the other stylings you made in this project.

File: css/style.css [hunk 3]
```diff
@@ -378,18 +384,651 @@ footer {
...
+    vertical-align: -0.5vw;
+  }
+
+  .contact-form {
+    background-image: url('../assets/img/image-geometry_6.svg'), url('../assets/img/image-geometry_7.svg');
+    background-position: -5.3vw top, right bottom;
+    background-repeat: no-repeat, no-repeat;
+    background-size: 24.3vw 18.26vw, 24.3vw 24.3vw;
+    margin: 0;
+    width: 100vw;
```
Comment (MahmoudBakr23) [inline css/style.css]: - [ ] Please apply what you learned from the above comment on this line here as well to fix the design overflow issue 🤗

File: css/style.css [hunk 4]
```diff
@@ -378,18 +384,651 @@ footer {
...
+    margin: 3.125vw 0;
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1.25vw;
+    text-align: center;
+    color: #fff;
+  }
+
+  .btn {
```
Comment (MahmoudBakr23) [inline css/style.css:518]: - [ ] Please make sure to follow the given design in all aspects such as typography, font size, font weight, colors, images, and spacings (margin and padding). Kindly fix this button's border to match the given design.
### **Your design**
Screenshot
### **[Original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9)**
Screenshot1

File: css/style.css [hunk 5]
```diff
@@ -378,18 +384,651 @@ footer {
...
+
+  .geometry {
+    background-image: url('../assets/img/image-geometry_3.png');
+    background-position: top right;
+    background-repeat: no-repeat;
+    background-size: 48.75vw 61vw;
+    flex: 1 1 50vw;
+  }
+
+  .main-text {
```
Comment (MahmoudBakr23) [inline css/style.css:474]: - [ ] Great work applying the background image for this section. But, there's missing one. Please refer to the comment above about following the given design and look at the images attached there. Also, kindly refer to the design template and make the corresponding changes.

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @smunozmo 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct 💯 
- Descriptive PR headline, summary, and commit messages 💯 
- Good use of Semantic HTML 💯 
- Descriptive README file 💯 

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 2 above)
Comment (smunozmo) [reply css/style.css]: Thank you, this helped me a lot! ✌️

File: index.html [hunk 6]
```diff
@@ -13,140 +13,188 @@
...
+                <li>Portfolio
+                    <div class="line-portfolio"></div>
+                </li>
+                <li>About
+                    <div class="line-about"></div>
+                </li>
+                <li>Contact
+                    <div class="line-contact"></div>
+                </li>
+            </ul>
```
Comment (Davidosky007) [inline index.html:30]: [OPTIONAL]
I strongly suggest you make each link an `a` tag as this is best practice

File: index.html (hunk 1 above)
Comment (Davidosky007) [reply index.html]: fixed 👍

File: css/style.css (hunk 2 above)
Comment (Davidosky007) [reply css/style.css]: fixed 👍

File: css/style.css (hunk 3 above)
Comment (Davidosky007) [reply css/style.css]: fixed 👍

File: css/style.css (hunk 4 above)
Comment (Davidosky007) [reply css/style.css:518]: fixed 👍

File: css/style.css (hunk 5 above)
Comment (Davidosky007) [reply css/style.css:474]: fixed 👍

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @smunozmo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏
* I left you some OPTIONAL changes below on making your links an `a` tag (social links, nav links)
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by smunozmo** [comment]
@MahmoudBakr23 @Davidosky007 thank you guys!

**Pull request merged**